tests/model_hub_tests/transformation_tests/generate_ref_diffs.py

```diff
@@ -0,0 +1,91 @@
+'''
+Use this script if you need to regenerate reference diffs for each model
+to test SDPAToPA transformation.
+
+The script will produce sdpa2pa_ref_diff.txt (or sdpa2pa_ref_diff_ce.txt
+if using cache-eviction) containing a map in the
+following format with nodes number changes for each model:
+
+pa_reference_map = {
+	"hf-internal-testing/tiny-random-LlamaForCausalLM" : {
+		"PagedAttentionExtension" : 2,
+		"ScaledDotProductAttention" : -2,
+		"Parameter" : 7,
+		"ReadValue" : -4,
+		"Assign" : -4,
+	},
+	"hf-internal-testing/tiny-random-CohereForCausalLM" : {
+		"PagedAttentionExtension" : 2,
+		"ScaledDotProductAttention" : -2,
+		"Parameter" : 7,
+		"ReadValue" : -4,
+		"Assign" : -4,
+	},
+    .
+    .
+    .
```

Comment:
?

Reply:
This is like "...", but vertical :)
Just to show that there're other models present in the resulting map.